kvk-api/app.js

```diff
@@ -0,0 +1,77 @@
+const fs = require('fs');
+const fetch = require("node-fetch")
+
+const express = require('express')
+const app = express()
+const port = process.env.API_PORT || 3001
+
+const redis = require("async-redis");
+const redis_kvk_cache = redis.createClient(
+    process.env.REDIS_KVK_CACHE_PORT || 6379,
+    process.env.REDIS_KVK_CACHE_HOST || "localhost"
+);
+
+redis_kvk_cache.on('connect', function() {
+    console.log('Redis client connected');
+});
+redis_kvk_cache.on("error", function(error) {
+    console.error(error);
+});
+
+const url = process.env.KVK_API_URL || "api.kvk.nl"
+const endpoint = process.env.KVK_API_ENDPOINT || "/api/v2/testprofile/companies"
+const api_key_name = process.env.KVK_API_KEY
+if (api_key_name) {
+    var api_key = null
+    try {
+        api_key = fs.readFileSync('/run/secrets/' + api_key_name, 'utf8');
+        console.log('API key succesfully loaded');
+    } catch(e) {
+        console.error('API key not found');
+    }
+}
+
+async function fetchApi (endpoint, api_key) {
+    var headers = {};
+    if (api_key) {
+        headers = {"apikey": api_key}
+    }
+    return fetch("https://" + endpoint, {headers: headers})
+        .catch(err => { console.error('caught', err.message); return {"error": 1} })
+        .then(data => data.json())
+}
+
+function productionCheck(req, res, next) {
+    if (process.env.NODE_ENV='production') {
```

Comment:
What is the purpose of using this middleware? (I mean, it's obvious it checks for production mode, but how does that help us?)

Reply:
It is to prevent serving the build folder when there is none during development. Could probably be cleaner, but it works :)

Reply:
I'd cut it out then, it adds an if condition at runtime for every request, even in production, right?

Reply:
Sure, I will remove it